### PR TITLE
add opera-disp-sandbox

### PR DESCRIPTION
**Pull request by jtherrmann**
TODO:

- [x] Should we deploy from `develop`?
- [x] Should we have separate test and prod deployments, both in the Tools Sandbox account? Or just have one deployment for now until we decide where these deployments will live long-term?
- [x] https://github.com/ASFHyP3/hyp3/pull/2501
- [x] Revert some development tweaks (see `TODO` in code)
- [x] Changelog
- [x] https://github.com/ASFHyP3/OPERA-DISP-TMS/pull/36

### CODE REVIEW

File: .github/workflows/deploy-opera-disp-sandbox.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+name: Deploy OPERA-DISP Sandbox Stack to AWS
+
+on:
+  push:
+    branches:
+      - opera-disp-sandbox
```
Comment (jtherrmann) [inline .github/workflows/deploy-opera-disp-sandbox.yml:6]:
```suggestion
      - develop
```

File: .github/workflows/deploy-opera-disp-sandbox.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        include:
+          - environment: hyp3-opera-disp-sandbox
+            domain: hyp3-opera-disp-sandbox.asf.alaska.edu
+            template_bucket: cf-templates-1hz9ldhhl4ahu-us-west-2
+            image_tag: test
+            product_lifetime_in_days: 14
+            default_credits_per_user: 0
+            default_application_status: APPROVED
+            cost_profile: DEFAULT
+            deploy_ref: refs/heads/opera-disp-sandbox
```
Comment (jtherrmann) [inline .github/workflows/deploy-opera-disp-sandbox.yml:25]:
```suggestion
            deploy_ref: refs/heads/develop
```

**Review by jtherrmann — COMMENTED** [review]
Should we deploy from `develop` like a regular test deployment?

File: .github/workflows/deploy-opera-disp-sandbox.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            job_files: >-
+              job_spec/INSAR_ISCE_BURST.yml
```
Comment (jtherrmann) [inline .github/workflows/deploy-opera-disp-sandbox.yml:27]:
```suggestion
              job_spec/OPERA_DISP_TMS.yml
```

**Review by jacquelynsmale — APPROVED** [review]

**Review by jacquelynsmale — APPROVED** [review]

**Pull request merged**